config/locales/views/en.yml

```diff
@@ -418,6 +421,46 @@ en:
         update: Update
       update:
         success: Archive settings were successfully updated.
+    support_notices:
+      confirm_delete:
+        delete: Delete Support Notice
+        submit: Yes, Delete Support Notice
+        warning_html: Are you sure you want to <strong><em>delete</em></strong> this support notice?
+      edit:
+        page_heading: Edit Support Notice
+      index:
+        page_heading: Support Notices
+      navigation:
+        index: Support Notices
+        new: New Support Notice
+      new:
+        page_heading: New Support Notice
+      show:
+        page_heading: Support Notice
+      support_notice:
+        active: Active
+        confirm_delete: Are you sure you want to delete this support notice?
+        delete: Delete
+        edit: Edit
+      support_notice_form:
+        actions:
+          cancel: Cancel
+          create: Create Support Notice
+          update: Update Support Notice
+        active: Active
+        landmark:
+          actions: Actions
+          content: Support Notice
+        legend:
+          actions: Actions
+          content: Support Notice
+        required_information: Required Information
+        text: Support Notice Text
+        type:
+          caution: Caution
+          error: Error
+          label: Support Notice Type
```

Comment:
Here and for Support Notice Text I think we do sentence case for these form labels now?